gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/sampling/functions/FilterVerticesWithDegreeOtherThanGiven.java

```diff
@@ -27,6 +27,21 @@
  */
 public class FilterVerticesWithDegreeOtherThanGiven implements UnaryGraphToGraphOperator {
 
+  /**
+   * Key of degree property
+   */
+  private static final String DEGREE_PROPERTY_KEY = VertexDegree.BOTH.getName();
```

Comment:
Duplicate constants. Use constants from SamplingAlgorithm class or add 3 string params to the constructor of this class to define them at instance creation.